### PR TITLE
CLDR-16122 v43 BRS: Merge v43 vetted data

### PR DESCRIPTION
**Pull request by srl295**
CLDR-16122

ALLOW_MANY_COMMITS=true
DISABLE_JIRA_ISSUE_MATCH=true

### CODE REVIEW

**Comment by srl295** [comment]
```shell
CLDR {
  TestPersonNameFormatter {
    TestAll (6.930s) Passed
    TestBasic (0.002s) Passed
    TestCheckForErrorsAndGetLocales (0.001s) Passed
    TestCheckForeignSpaceReplacement (0.001s) Passed
    TestCheckInitials (0.000s) Passed
    TestCheckNameOrderLocales (0.001s) Passed
    TestCheckPatterns (0.000s) Passed
    TestCheckPersonNames (0.000s) Passed
    TestCheckSampleNames (0.000s) Passed
    TestEnglishComma (0.001s) Passed
    TestExampleDependencies {
      Error: (TestPersonNameFormatter.java:415) : en example for {title} {given} {given2} {surname} {generation}, {credentials}: expected "〖<i>🟨 Native name and script:</i>〗〖❬Zendaya❭〗〖❬Irene❭ ❬Adler❭〗〖❬Mary Sue❭ ❬Hamish❭ ❬Watson❭〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬Sinbad❭〗〖❬Käthe❭ ❬Müller❭〗〖❬Zäzilia❭ ❬Hamish❭ ❬Stöber❭〗〖❬Prof. Dr.❭ ❬Ada Cornelia❭ ❬César Martín❭ ❬von Brühl❭ ❬Jr❭, ❬MD DDS❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Єва❭ ❬Марія❭ ❬Шевченко❭〗〖❬太郎トーマス山田❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬Zendaya❭〗〖❬Irene❭ ❬Adler❭〗〖❬Mary Sue❭ ❬Hamish❭ ❬Watson❭〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬Sinbad❭〗〖❬Käthe❭ ❬Müller❭〗〖❬Zäzilia❭ ❬Hamish❭ ❬Stöber❭〗〖❬Prof. Dr.❭ ❬Ada Cornelia❭ ❬César Martín❭ ❬von Brühl❭ ❬Jr❭, ❬MD DDS❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬проф.❭ ❬Анна Марія❭ ❬Богдан-Зиновій❭ ❬Петренко❭ ❬доц.❭〗〖❬恵子グレース佐藤ジュニアさん❭〗"
      Error: (TestPersonNameFormatter.java:415) : en example for {title} {given} {given2} {surname} {generation}, {credentials}: expected "〖<i>🟨 Native name and script:</i>〗〖❬Zendaya❭〗〖❬IRENE2❭ ❬Adler❭〗〖❬Mary Sue❭ ❬Hamish❭ ❬Watson❭〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬Sinbad❭〗〖❬Käthe❭ ❬Müller❭〗〖❬Zäzilia❭ ❬Hamish❭ ❬Stöber❭〗〖❬Prof. Dr.❭ ❬Ada Cornelia❭ ❬César Martín❭ ❬von Brühl❭ ❬Jr❭, ❬MD DDS❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Єва❭ ❬Марія❭ ❬Шевченко❭〗〖❬太郎トーマス山田❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬Zendaya❭〗〖❬IRENE2❭ ❬Adler❭〗〖❬Mary Sue❭ ❬Hamish❭ ❬Watson❭〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬Sinbad❭〗〖❬Käthe❭ ❬Müller❭〗〖❬Zäzilia❭ ❬Hamish❭ ❬Stöber❭〗〖❬Prof. Dr.❭ ❬Ada Cornelia❭ ❬César Martín❭ ❬von Brühl❭ ❬Jr❭, ❬MD DDS❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬проф.❭ ❬Анна Марія❭ ❬Богдан-Зиновій❭ ❬Петренко❭ ❬доц.❭〗〖❬恵子グレース佐藤ジュニアさん❭〗"
    } (0.146s) FAILED (2 failure(s))
    TestExampleGenerator {
      Error: (TestPersonNameFormatter.java:415) : en example for {title} {given} {given2} {surname} {generation}, {credentials}: expected "〖<i>🟨 Native name and script:</i>〗〖❬Zendaya❭〗〖❬Irene❭ ❬Adler❭〗〖❬Mary Sue❭ ❬Hamish❭ ❬Watson❭〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬Sinbad❭〗〖❬Käthe❭ ❬Müller❭〗〖❬Zäzilia❭ ❬Hamish❭ ❬Stöber❭〗〖❬Prof. Dr.❭ ❬Ada Cornelia❭ ❬César Martín❭ ❬von Brühl❭ ❬Jr❭, ❬MD DDS❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Єва❭ ❬Марія❭ ❬Шевченко❭〗〖❬太郎トーマス山田❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬Zendaya❭〗〖❬Irene❭ ❬Adler❭〗〖❬Mary Sue❭ ❬Hamish❭ ❬Watson❭〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬Sinbad❭〗〖❬Käthe❭ ❬Müller❭〗〖❬Zäzilia❭ ❬Hamish❭ ❬Stöber❭〗〖❬Prof. Dr.❭ ❬Ada Cornelia❭ ❬César Martín❭ ❬von Brühl❭ ❬Jr❭, ❬MD DDS❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬проф.❭ ❬Анна Марія❭ ❬Богдан-Зиновій❭ ❬Петренко❭ ❬доц.❭〗〖❬恵子グレース佐藤ジュニアさん❭〗"
      Error: (TestPersonNameFormatter.java:415) : en example for {surname-monogram-allCaps}{given-informal-monogram-allCaps}: expected "〖<i>🟨 Native name and script:</i>〗〖❬Z❭〗〖❬AI❭〗〖❬WM❭〗〖❬WB❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬S❭〗〖❬MK❭〗〖❬SZ❭〗〖❬VN❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬ШЄ❭〗〖❬太❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬Z❭〗〖❬AI❭〗〖❬WM❭〗〖❬WB❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬S❭〗〖❬MK❭〗〖❬SZ❭〗〖❬VN❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬ПМ❭〗〖❬恵❭〗"
      Error: (TestPersonNameFormatter.java:415) : ja example for {given} {given2} {surname} {generation}{title}: expected "〖<i>🟨 Native name and script:</i>〗〖❬慎太郎❭〗〖❬一郎安藤❭〗〖❬太郎トーマス山田❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬アルベルト❭・❬アインシュタイン❭〗〖❬英子❭・❬ソフィア❭・❬内田ドクター❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖❬Єва❭ ❬Марія❭ ❬Шевченко❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬慎太郎❭〗〖❬一郎安藤❭〗〖❬太郎トーマス山田❭〗〖❬恵子グレース佐藤ジュニアさん❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬マイケル❭〗〖❬アルベルト❭・❬アインシュタイン❭〗〖❬セシリア❭・❬ローズ❭・❬ブラウン❭〗〖❬ジェニファー❭・❬ソフィア❭・❬フォン・スミス❭・❬ジュニア博士❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖❬проф.❭ ❬Анна Марія❭ ❬Богдан-Зиновій❭ ❬Петренко❭ ❬доц.❭〗"
      Error: (TestPersonNameFormatter.java:415) : th example for {title} {given} {given2} {surname} {credentials}: expected "〖<i>🟨 Native name and script:</i>〗〖❬ธนา❭〗〖❬ไอริณกล้าหาญ❭〗〖❬วีระพลชัยยศพิชิตชัย❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬ศ.ดร.โสพลชัยฤทธิ์ณนคร❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖❬Єва❭ ❬Марія❭ ❬Шевченко❭〗〖❬太郎トーマス山田❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬ธนา❭〗〖❬ไอริณกล้าหาญ❭〗〖❬มานีชัยยศพิชิตชัย❭〗〖❬คุณปรีชากล้าหาญแสงระวี❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬ซินแบด❭〗〖❬เคเทอ❭ ❬มึลเลอร์❭〗〖❬ซาซิเลีย❭ ❬ฮามิช❭ ❬สโตเบอร์❭〗〖❬ศ.ดร.❭ ❬เอดา คอร์เนเลีย❭ ❬เซซาร์ มาร์ติน❭ ❬วอน บรืล❭ ❬พ.บ. ท.บ.❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Mr.❭ ❬Bertram Wilberforce❭ ❬Henry Robert❭ ❬Wooster❭ ❬Jr❭, ❬MP❭〗〖❬проф.❭ ❬Анна Марія❭ ❬Богдан-Зиновій❭ ❬Петренко❭ ❬доц.❭〗〖❬恵子グレース佐藤ジュニアさん❭〗"
    } (0.728s) FAILED (4 failure(s))
    TestFallbackFormatter (0.000s) Passed
    TestFields (0.001s) Passed
    TestForeignNonSpacingNames (0.002s) Passed
    TestFormatAll (0.001s) Passed
    TestInconsistentModifiers (0.007s) Passed
    TestInconsistentNameValues (0.000s) Passed
    TestLabels (0.000s) Passed
    TestLiteralTextElision (0.001s) Passed
    TestLiteralTextElisionNoSpaces (0.000s) Passed
    TestLiteralTextElisionSpaces (0.001s) Passed
    TestNamePatternParserRountripsValidPattern (0.000s) Passed
    TestNamePatternParserThrowsWhenInvalidPatterns (0.001s) Passed
    TestSupplementalConsistency (2.162s) Passed
    TestWithCLDR {
      Warning: (TestPersonNameFormatter.java:178) To see the contents of the English patterns, use -DTestPersonNameFormatter.SHOW
    } (0.001s) ALERT (1 warning(s))
    testAttributeValueOrder (0.001s) Passed
    testEmptyFsrWrite (0.662s) Passed
    testInitials {
      Error: (TestPersonNameFormatter.java:415) : en example for {given-initial} {given2-initial} {surname}: expected "〖<i>🟨 Native name and script:</i>〗〖❬Z.❭〗〖❬I.❭ ❬Adler❭〗〖❬M. S.❭ ❬H.❭ ❬Watson❭〗〖❬B. W.❭ ❬H. R.❭ ❬Wooster❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬S.❭〗〖❬K.❭ ❬Müller❭〗〖❬Z.❭ ❬H.❭ ❬Stöber❭〗〖❬A. C.❭ ❬C. M.❭ ❬von Brühl❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬Є.❭ ❬М.❭ ❬Шевченко❭〗〖❬太郎山田❭〗", got "〖<i>🟨 Native name and script:</i>〗〖❬Z.❭〗〖❬I.❭ ❬Adler❭〗〖❬M. S.❭ ❬H.❭ ❬Watson❭〗〖❬B. W.❭ ❬H. R.❭ ❬Wooster❭〗〖<i>🟧 Foreign name and native script:</i>〗〖❬S.❭〗〖❬K.❭ ❬Müller❭〗〖❬Z.❭ ❬H.❭ ❬Stöber❭〗〖❬A. C.❭ ❬C. M.❭ ❬von Brühl❭〗〖<i>🟥 Foreign name and script:</i>〗〖❬проф.❭ ❬А. М.❭ ❬Б.❭ ❬Петренко❭ ❬доц.❭〗〖❬恵子佐藤さん❭〗"
    } (0.075s) FAILED (1 failure(s))
    testTransliteratorName {
      Error: (TestAll.java:178) (TestAll.java:178) java.lang.IllegalArgumentException: Illegal ID Olck-Latn
java.lang.IllegalArgumentException: Illegal ID Olck-Latn
        at com.ibm.icu.text.TransliteratorIDParser.instantiateList(TransliteratorIDParser.java:439)
        at com.ibm.icu.text.Transliterator.getInstance(Transliterator.java:1603)
        at com.ibm.icu.text.Transliterator.getInstance(Transliterator.java:1576)
        at org.unicode.cldr.util.CLDRTransforms.getScriptTransform(CLDRTransforms.java:935)
        at org.unicode.cldr.unittest.TestPersonNameFormatter.testTransliteratorName(TestPersonNameFormatter.java:989)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:577)
        at com.ibm.icu.dev.test.TestFmwk$MethodTarget.execute(TestFmwk.java:429)
        at com.ibm.icu.dev.test.TestFmwk$Target.run(TestFmwk.java:367)
        at com.ibm.icu.dev.test.TestFmwk$ClassTarget.execute(TestFmwk.java:486)
        at com.ibm.icu.dev.test.TestFmwk$Target.run(TestFmwk.java:367)
        at com.ibm.icu.dev.test.TestFmwk$ClassTarget.execute(TestFmwk.java:486)
        at com.ibm.icu.dev.test.TestFmwk$Target.run(TestFmwk.java:367)
        at com.ibm.icu.dev.test.TestFmwk.runTests(TestFmwk.java:645)
        at com.ibm.icu.dev.test.TestFmwk.run(TestFmwk.java:593)
        at org.unicode.cldr.unittest.TestAll.runTests(TestAll.java:178)
        at org.unicode.cldr.unittest.TestShim.TestAll(TestShim.java:21)
        at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
        at java.base/java.lang.reflect.Method.invoke(Method.java:577)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:725)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
        at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:214)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:210)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:135)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:66)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:220)
        at org.junit.platform.launcher.core.DefaultLauncher.lambda$execute$6(DefaultLauncher.java:188)
        at org.junit.platform.launcher.core.DefaultLauncher.withInterceptedStreams(DefaultLauncher.java:202)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:181)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:128)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invokeAllTests(JUnitPlatformProvider.java:150)
        at org.apache.maven.surefire.junitplatform.JUnitPlatformProvider.invoke(JUnitPlatformProvider.java:124)
        at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
        at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)

    } (6.105s) FAILED (1 failure(s))
  } (16.833s) FAILED (8 failure(s), 1 warning(s))
    Warning: (TestAll.java:178) Test org.unicode.cldr.unittest.TestLdml2ICU not found or not valid.
    Warning: (TestAll.java:178) Test org.unicode.cldr.unittest.TestLocaleCanonicalizer not found or not valid.
    Warning: (TestAll.java:178) Test org.unicode.cldr.unittest.TestNumbers not found or not valid.
} (20.955s) FAILED (8 failure(s), 4 warning(s), 436 test(s) skipped)

Error summary:
CLDR/TestPersonNameFormatter/TestExampleDependencies
CLDR/TestPersonNameFormatter/TestExampleGenerator
CLDR/TestPersonNameFormatter/testInitials
CLDR/TestPersonNameFormatter/testTransliteratorName
```

@macchiati do these belong as unit tests or should they be console checks, locale-specific? I find the output very hard to understand as a unit test.  no details about which locale, etc.

**Comment by macchiati** [comment]
The extra TestPersonNameFormatter tests were from when we were redesigning the PN structure, so I'll look at them and see what can be repurposed.

**Comment by srl295** [comment]
> The extra TestPersonNameFormatter tests were from when we were redesigning the PN structure, so I'll look at them and see what can be repurposed.

see #2672 - I propose to skip these until they can be reevaluated

**Comment by macchiati** [comment]
Just made a PR to resolve the TestPN failures.

**Comment by srl295** [comment]
#2669  needs review still

**Comment by srl295** [comment]
I'm going to mark this as ready to review because I'm hoping it will be passing its tests soon

**Comment by srl295** [comment]
> jira-ticket — Commit 62a5b1e is for CLDR-16122, but the PR is for CLDR-16239

I changed the PR to match the first commit's jira ID CLDR-16239

but this may be an issue.

**Comment by srl295** [comment]
@pedberg-icu @btangmu @macchiati  good possibility this will pass… can i get a review?

File: common/main/ar.xml [hunk 1]
```diff
@@ -317,6 +317,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<language type="li">الليمبورغية</language>
 			<language type="lil">الليلويتية</language>
 			<language type="lkt">لاكوتا</language>
+			<language type="lmo" draft="contributed">اللومبردية</language>
```
Comment (pedberg-icu) [inline common/main/ar.xml:320]: Curious that this path got added in limited submission

File: common/main/bg.xml [hunk 2]
```diff
@@ -305,6 +305,7 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 			<language type="li">лимбургски</language>
 			<language type="lil">лилоует</language>
 			<language type="lkt">лакота</language>
+			<language type="lmo" draft="contributed">ломбардски</language>
```
Comment (pedberg-icu) [inline common/main/bg.xml:308]: Hmm, added as in Arabic. Were we collecting names for this?

**Review by macchiati — APPROVED** [review]
I think this is fine for now. The transforms changes are not quite right, but I'll file a ticket to straighten out later.

File: common/main/bg.xml (hunk 2 above)
Comment (srl295) [reply common/main/bg.xml:308]: `lmo` is a new locale for basic. did it get special handling?

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRTransforms.java [hunk 3]
```diff
@@ -931,6 +932,11 @@ public static Transliterator getScriptTransform(final String script) {
         case "Hans":
             id = "Han-Latn";
             break;
+        case "Olck":
+            id = "sat_Olck-sat_FONIPA"; // Latin IPA
+            break;
+        default:
+            id = script + "-Latn";
```
Comment (pedberg-icu) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRTransforms.java:939]: Ooh, nice to add default

**Review by pedberg-icu — APPROVED** [review]

File: common/main/bg.xml (hunk 2 above)
Comment (macchiati) [reply common/main/bg.xml:308]: Missing values (needed for CLDR target coverage) could be added. If we added a new language type, then that created a new Missing item. If that was done at the end of submission, then like the beaufort paths, we need to subtract from coverage (AND not do it in the future!)

Comment (srl295) [reply common/main/bg.xml:308]: Investigated in https://unicode-org.atlassian.net/browse/CLDR-16347  - it was due to being missing.

Comment (srl295) [reply common/main/bg.xml:308]: Same with Tatar

**Pull request merged**